test/lit/passes/dae-gc.wast

```diff
@@ -69,30 +69,29 @@
    (unreachable)
   )
  )
- ;; a function that gets an rtt that is never used. we cannot create a local for
+ ;; a function that gets a non-nullable reference that is never used. we cannot create a local for
  ;; that parameter, as it is not defaultable, so do not remove the parameter.
- ;; CHECK:      (func $get-rtt (param $0 (rtt ${}))
+ ;; TODO: We should be able to create a local for it after #4824 is resolved.
```

Comment:
it looks from the output like we do optimize here. the argument is removed and a local is added, as nullable.